pkg/cloudprovider/provider/gce/provider.go

```diff
@@ -112,6 +115,13 @@ func (p *Provider) Validate(spec clusterv1alpha1.MachineSpec) error {
 	if cfg.zone == "" {
 		return newError(common.InvalidConfigurationMachineError, errInvalidZone)
 	}
+
```

Comment:
Maybe we should also prevent `IPv6` (only) network family here, as it is not supported by GCP (yet).

Reply:
Sure. Or we could just remove the option from the list?

Reply:
Since it is aimed to be common across all cloud providers, let's keep it. May be useful for some that would already support IPv6-only.

Reply:
Added validation for each family.